animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -130,6 +130,9 @@ cdef class Mesh:
     def getUVMapNames(self):
         return list(self.uvMaps.keys())
 
+    def getUVMapCos(self, str uvMapName):
```

Comment:
What does `Cos` denote here?

Reply:
UV coordinates.

Reply:
I see. It is not immediately clear to me, so I would consider changing it.

Reply:
Okay. Should I rename it as "UVMapVectors"?

Reply:
Ok. Or maybe, `getUVMapPositions`.

Reply:
Okay. In the next commit, I'll change that.